lib/bugsnag/payload.ex

```diff
@@ -62,23 +61,29 @@ defmodule Bugsnag.Payload do
         %{
           file: "unknown",
           lineNumber: 0,
-          method: "#{ module }.#{ function }#{ format_args(args) }"
+          method: Exception.format_mfa(module, function, args)
         }
       ({ module, function, args, [file: file, line: line_number] }) ->
         %{
-          file: file |> List.to_string,
+          file: List.to_string(file),
           lineNumber: line_number,
-          method: "#{ module }.#{ function }#{ format_args(args) }"
+          inProject: String.starts_with?(file, "web"),
```

Comment:
This looks like it's specific to the current structure of Phoenix applications, and doesn't take into consideration that parts of even a Phoenix app (but also other non-Phoenix projects) might live inside `lib`.